scripts/import/import_all_sv_data.pl

```diff
@@ -274,7 +274,7 @@ =head1 CONTACT
 # Post processing for mouse annotation (delete duplicated entries in structural_variation_sample)
 post_processing_annotation() if ($species =~ /mouse|mus/i);
 post_processing_feature();
-post_processing_sample();
+post_processing_sample() if ($source_name eq 'DGVa');
```

Comment:
We also want to run it for mouse.

Reply:
Yeah, then I realized the 2 queries we are running are both for `DGVa` studies -

https://github.com/Ensembl/ensembl-variation/blob/d34d25e481f22c11705b23a1aded34feb8434010/scripts/import/import_all_sv_data.pl#L1777
https://github.com/Ensembl/ensembl-variation/blob/d34d25e481f22c11705b23a1aded34feb8434010/scripts/import/import_all_sv_data.pl#L1794

So, I think that condition is redundant.

Reply:
Keeping the condition logic as it is for now in line with offline discussion